<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>insertDialog</class>
 <widget class="QDialog" name="insertDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加信息</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>部门</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="pos_lineEdit"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>楼层和房间号</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="sex_lineEdit"/>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="building_lineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="college_label">
       <property name="text">
        <string>电话</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="sex_label">
       <property name="text">
        <string>性别</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>薪资</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="name_lineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="name_label">
       <property name="text">
        <string>姓名</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="number_label">
       <property name="text">
        <string>工号</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="salary_lineEdit"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="tel_lineEdit"/>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="job_lineEdit"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="sid_ineEdit"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="age_label">
       <property name="text">
        <string>工位</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>职位</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="part_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="insert_pushButton">
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
